neclib/devices/ad_converter/cpz3177.py

```diff
@@ -7,6 +7,57 @@
 
 
 class CPZ3177(ADConverter):
+
+    """a/d converter, which can convert by 32 or 64 channels.
+
+    Notes
+    -----
+    Configuration items for this device:
+
+    rsw_id : {0, 1, ..., 16} or {"0", "1", ..., "9", "A", ..., "F"}
+        Board identifier. This should be set to the same value as the rotary switch
+        "RSW1" mounted on the side of the board. The board is shipped with default RSW1
+        setting of 0. This ID would be non-zero, when multiple PCI board of same model
+        are mounted on a single FA (Factory Automation) controller.
+
+    ave_num : int
+        Sampling data number to calculate average of data. This source code not only
+        measure the actual voltage, but also calculate the average of data
+        in order to measure a faint voltage. It should not be define too large number
+        against sampling frequency.
+
+    smpl_freq : int
+        Sampling frequency. Number of measuring data per 1 second.
+        It should not be too small number against number of data for
+        calculating average.
```

Comment:
```suggestion
        Sampling frequency; number of data obtained in 1 second. The value should not
        be too small against ``ave_num``.
```

`ave_num` だけでなく `smpl_freq` の値にも気を遣う必要がある、という理解でいいでしょうか？
理由があれば追記してほしいです (e.g., せっかく高いサンプリング周波数でデータを得ているのに時間分解能が悪くなる、メモリを食い潰す、etc.)。

Reply:
```ave_num```, ```smpl_freq```共に互いに大きすぎ・小さすぎに気を使う理由としてはSuggestionの通りで他に特にありませんのでそのように修正しました。実際に現地でテストした時に値の変化に対応できていないような挙動を示していたので記述しています。